networkx/drawing/layout.py

```diff
@@ -97,7 +102,10 @@ def random_layout(G, center=None, dim=2, seed=None):
     --------
     >>> G = nx.lollipop_graph(4, 3)
     >>> pos = nx.random_layout(G)
-
+    >>> # supress the returned dict and store on the graph directly
+    >>> _ = nx.random_layout(G, seed=42, store_pos_as="pos")
```

Comment:
```
-    >>> _ = nx.random_layout(G, seed=42, store_pos_as="pos")
+    >>> nx.random_layout(G, seed=42, store_pos_as="pos")
```

Reply:
Removing the `_ =` will cause the repl to display the returned `dict` from the method (since it always returns a `dict`) and I'd like to keep the visual clarify of not having the `dict` displayed immediately to show that the new attribute is on the graph.